diff --git a/lua/core/options.lua b/lua/core/options.lua
index ce2a54c..f2eaa00 100644
--- a/lua/core/options.lua
+++ b/lua/core/options.lua
@@ -17,7 +17,7 @@ astronvim.vim_opts(astronvim.user_plugin_opts("options", {
     number = true, -- Show numberline
     preserveindent = true, -- Preserve indent structure as much as possible
     pumheight = 10, -- Height of the pop up menu
-    relativenumber = true, -- Show relative numberline
+    relativenumber = false, -- Show relative numberline
     scrolloff = 8, -- Number of lines to keep above and below the cursor
     shiftwidth = 2, -- Number of space inserted for indentation
     showmode = false, -- Disable showing modes in command line
diff --git a/lua/core/plugins.lua b/lua/core/plugins.lua
index 330a466..55e189a 100644
--- a/lua/core/plugins.lua
+++ b/lua/core/plugins.lua
@@ -1,4 +1,5 @@
 local astro_plugins = {
+  ['pantharshit00/vim-prisma'] = {},
   -- Plugin manager
   ["wbthomason/packer.nvim"] = {
     setup = function()
